jumanji/training/setup_train.py

```diff
@@ -134,7 +152,7 @@ def _setup_random_policy(  # noqa: CCR001
     cfg: DictConfig, env: Environment
 ) -> RandomPolicy:
     assert cfg.agent == "random"
-    if cfg.env.name == "bin_pack":
+    if cfg.env.name == "bin_pack" or cfg.env.name == "upgraded_bin_pack":
         assert isinstance(env.unwrapped, BinPack)
```

Comment:
Shouldn't it be a a constrained binpack class in this case?

Reply:
done ✅